cmd/system-probe/modules/all_linux.go

```diff
@@ -3,7 +3,7 @@
 // This product includes software developed at Datadog (https://www.datadoghq.com/).
 // Copyright 2016-present Datadog, Inc.
 
-//go:build linux && !arm64
```

Comment:
Go build tags are usually positive assertions, so let's use a `glibc` build tag instead and add it to the list of default build tags.